src/main/java/com/autotune/analyzer/serviceObjects/verification/validators/KubernetesElementsValidator.java

```diff
@@ -57,19 +58,20 @@ public boolean isValid(UpdateResultsAPIObject updateResultsAPIObject, Constraint
 
             // Check if Kubernetes Object Namespaces is matched
             if (kruizeObject.getExperimentType().equals(AnalyzerConstants.ExperimentType.NAMESPACE)) {
-                // it's a namespace experiment type then type, name, namespace should not be present
-
-                String kubeObjTypeInResultData = resultData.getKubernetes_objects().get(0).getType();
-                String kubeObjNameInResultsData = resultData.getKubernetes_objects().get(0).getName();
-                String kubeObjNameSpaceInResultsData = resultData.getKubernetes_objects().get(0).getNamespace();
-
-                if (kubeObjTypeInResultData != null && kubeObjNameInResultsData != null && kubeObjNameSpaceInResultsData != null) {
-                    kubeObjsMisMatch = true;
-                    errorMsg = errorMsg.concat(
-                            String.format(
-                                    AnalyzerErrorConstants.AutotuneObjectErrors.MISSING_NAMESPACE_SPECIFIC_UPDATE_RESULTS_FIELDS,
-                                    expName
-                            ));
+                // Validate the absence of type, name and namespace objects
+                for (K8sObject obj : resultData.getKubernetes_objects()) {
```

Comment:
It seems this change is needed in multiple parts of the method, so applying it in just one place wouldn’t be appropriate. For now, let’s avoid expanding the scope of this PR. Please add a comment here noting the assumption we're making (i.e., that there is only one Kubernetes object). 
We’ll address the broader changes in a separate PR.